### ISSUE TITLE
Repair schedule in UTC timezone

### ISSUE THREAD

**Issue by dhillonarvi** [thelastpickle/cassandra-reaper#942]
[Project board link](https://github.com/orgs/k8ssandra/projects/8/views/1?pane=issue&itemId=13951860)

Most of our servers operate in UTC time zone it would be great if we could just schedule repairs in UTC timezone and view logs in same TZ.

┆Issue is synchronized with this [Jira Story](https://datastax.jira.com/browse/REAP-127) by [Unito](https://www.unito.io)
┆Issue Number: REAP-127

**Comment by adejanovski** [comment]
Actually, we went into some efforts to display times in the local TZ in the UI 😅 

I guess we could have a setting added to make everything displayed in UTC.
I'd recommend to contribute that feature though if you need it quickly.

**Comment by eriksw** [comment]
It's **really** important to us that the times we see when when viewing and creating schedules match up with the times in logs.

At the very least, please make it obvious / explicitly documented in the UI that the times are in the viewer's browser time. Right now, when you're looking at the UI creating a schedule, there's no indication that the time isn't UTC, which is a really nasty surprise.

**Comment by adejanovski** [comment]
Fair enough. I guess we can indicate that the timezone is local by default and allow users to set the times to UTC in the UI.
